tests/test_smoketest.py

```diff
@@ -63,16 +66,30 @@ def test_smoketest(tmp_path, format):
     assert len(contents) == 1 + (2 * 2 * 4)  # Header row + 2 STPs * 2 sexes * 4 dates
     dates = set(row[5] for row in contents[1:])
     assert dates == {"2020-01-01", "2020-02-01", "2020-03-01", "2020-04-01"}
+
     # liver_disease
     with open(tmp_path / "measure_liver_disease.csv") as f:
         contents = list(csv.reader(f))
+    assert contents[0] == [
+        "has_chronic_liver_disease",
+        "population",
+        "value",
+        "date",
+    ]
```

Comment:
Could there be an assert that makes it clear how this one differs from the one-group version?  Does it differ, if it has no groupby?

Reply:
Ah yeah, good point. The tests are very much not great here. The difference is that in the "no group" version every patient is on their own line so the assert would be something like "number of rows = population X number of dates" and "population = 1 for every row".

This all started as a very basic smoke test (as the name implies) that the main entrypoint was functioning OK. Then when the measures stuff was added it was a quick and dirty way of adding some kind of testing for it then was always intended to be replaced with "proper testing" at some point and then never was :)